genesyscloud/responsemanagement_response/resource_genesyscloud_responsemanagement_response_test.go

```diff
@@ -48,6 +48,7 @@ func TestAccResourceResponseManagementResponseFooterField(t *testing.T) {
 		assetResource = "asset-resource"
 		fileName      = "yeti-img.png"
 		fullPath      = fmt.Sprintf("%s/%s", testFilesDir, fileName)
+		filePath      = "test_responseasset_data/yeti-img.png"
```

Comment:
I think we should start using `filepath.Join` for test cases like this. I'm pretty sure this wouldn't work on a windows machine because they use \ instead of / 

I had to deal with a bug in the CLI recently that was caused by manually built path strings (code I wrote) instead of using a package that would account for different operating systems.